File: learn/prog/input_script/input_script.php
<?php

// Example of execution: php input_script.ps1 -3.4 'Hello, world!' 6



$a = $argv;
$b = $argv[0];
$c = $argv[1];
$d = $argv[2];
$e = $argv[3];

echo PHP_EOL;
echo 'Arguments obtained (array): ' . implode(', ', $a) . PHP_EOL;
echo PHP_EOL;
echo 'Number of arguments: ' . count($a) . PHP_EOL;
echo PHP_EOL;
echo 'Script name: ' . $b . PHP_EOL;
echo PHP_EOL;
echo 'First argument: ' . $c . PHP_EOL;
echo 'Second argument: ' . $d . PHP_EOL;
echo 'Third argument: ' . $e . PHP_EOL;
echo PHP_EOL;
echo 'All arguments are strings. E.g. type of second argument: ' . gettype($d) . PHP_EOL;
echo PHP_EOL;

?>

File: learn/prog/input_script/input_script.py


# Example of execution: python3 input_script.py -3.4 'Hello, world!' 6

import sys

a = sys.argv
b = sys.argv[0]
c = sys.argv[1]
d = sys.argv[2]
e = sys.argv[3]

print(' ')
print('Arguments obtained (list):', a)
print(' ')
print('Number of arguments:', len(a))
print(' ')
print('Script name:', b)
print(' ')
print('First argument:', c)
print('Second argument:', d)
print('Third argument:', e)
print(' ')
print('All arguments are strings. E.g. type of second argument:', type(d))
print(' ')


